hypershift-operator/controllers/nodepool/kubevirt/kubevirt.go

```diff
@@ -280,6 +266,65 @@ func virtualMachineTemplateBase(nodePool *hyperv1.NodePool, bootImage BootImage)
 	return template
 }
 
+func virtualMachineInterfaceName(idx int) string {
+	return fmt.Sprintf("iface%d", idx+1)
```

Comment:
would it make sense to bring semantic meaning into this name ? 

Like, before it's name was `default` (for the default network). Now it will be named `iface1`...

Maybe put the name of the network attachment (after trimming the namespace) (if it is a secondary network) ?

Reply:
we cannot trim the namespace since it can collide, but we can serialize it

Reply:
Also it can happend that multiple interfaces use the same nad (I think).

Reply:
yes, indeed you can.